crates/sncast/src/response/serialize.rs

```diff
@@ -0,0 +1,45 @@
+use super::command::CommandResponse;
+use crate::response::cast_message::SncastMessage;
+use conversions::serde::serialize::CairoSerialize;
+use foundry_ui::Message;
+use foundry_ui::styling;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use serde_json::json;
+use starknet_types_core::felt::Felt;
+
+#[derive(Serialize, Deserialize, CairoSerialize, Clone, Debug, PartialEq)]
+pub struct SerializeResponse {
+    pub calldata: Vec<Felt>,
+}
+
+impl CommandResponse for SerializeResponse {}
+
+impl Message for SncastMessage<SerializeResponse> {
+    fn text(&self) -> String {
+        let calldata = &self
+            .command_response
+            .calldata
+            .iter()
+            .map(|felt| format!("{felt:?}"))
+            .collect::<Vec<String>>()
+            .join(", ");
+        let calldata = format!("[{calldata}]");
+
+        styling::OutputBuilder::new()
+            .success_message("Serialization completed")
```

Comment:
This is kind of redundant, I'd just show message as `Serialized calldata:\n{serialized}` (you can add some formatting)